packages/init/.eslintrc.cjs

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
```

Comment:
browser?

Reply:
Removed in this commit: https://github.com/prismicio/slice-machine/commit/e501e89ef5fd54cf4607e688e9cad4cdc7c58890